### DISCUSSION TITLE
Option to toggle food by default

### DISCUSSION THREAD

**Discussion by djlombar89** [mealie-recipes/mealie#4715] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When adding items to a shopping list, I have to toggle food every time. While this is made better with the update earlier this year to leave it toggled when consecutively adding an item, 99% of the time I want to add an item that is from my foods list so that I get autocomplete and category without having to enter additional information.

So anytime I go into my shopping list, I have to toggle it on.  I frequently think of an item, go in the app, and have to toggle it on before I can actually add my item.  It feels odd that a "note" is the primary item instead of the actual item.

### Please provide a concise description of the feature that would resolve your issue.

I would like to have the option to have food toggled by default.  Ideally as a user setting so that the behavior can match a user's preference rather than forcing default behavior.

### Please consider and list out some caveats or tradeoffs made in your design decision

If made a user setting, none that I can think of.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Probably something as simple as remembering your last choice will work for this. Similar to the previous update how it persists between items, but making it persist between sessions.

**Reply by djlombar89** [reply]
That sounds like a great solution as well!
